io.catenax.week_based_capacity_group/2.0.0/WeekBasedCapacityGroup.ttl

```diff
@@ -0,0 +1,357 @@
+
+:name a samm:Property ;
+   samm:preferredName "Capacity Group Name"@en ;
+   samm:description "Name of the capacity group."@en ;
+   samm:characteristic :CapacityGroupName ;
+   samm:exampleValue "Spark Plugs on drilling machine for car model XYZ" .
+
+:supplierLocations a samm:Property ;
+   samm:preferredName "Supplier Locations"@en ;
+   samm:description "Set of BPNS indicating the supplier sites that will be fulfilling the demands associated with this capacity group."@en ;
+   samm:characteristic :BPNSSetTrait ;
+   samm:exampleValue "BPNS8888888888XX" .
+
+:customer a samm:Property ;
+   samm:preferredName "Customer"@en ;
+   samm:description "The Business Partner Number (BPNL) of the party requesting materials from a supplier."@en ;
+   samm:characteristic :BPNLTrait ;
+   samm:exampleValue "BPNL8888888888XX" .
+
+:supplier a samm:Property ;
+   samm:preferredName "Supplier"@en ;
+   samm:description "The Business Partner Number (BPNL) of the party providing materials to a customer."@en ;
+   samm:characteristic :BPNLTrait ;
+   samm:exampleValue "BPNL6666666666YY" .
+
+:unitOfMeasure a samm:Property ;
+   samm:preferredName "Unit of Measure"@en ;
+   samm:description "Unit of measurement for capacity quantities."@en ;
+   samm:characteristic ext-quantity:ItemUnitEnumeration ;
+   samm:exampleValue "unit:piece"^^samm:curie .
+
+:linkedDemandSeries a samm:Property ;
+   samm:preferredName "Linked Demand Series"@en ;
+   samm:description "Set of demand series assigned to this capacity group."@en ;
+   samm:characteristic :LinkedDemandSeriesSet .
+
+:capacities a samm:Property ;
+   samm:preferredName "Capacities"@en ;
+   samm:description "A time series with week-based granularity along a given time period containing the capacity values."@en ;
+   samm:characteristic :CapacityTimeSeries .
+
+:changedAt a samm:Property ;
+   samm:preferredName "Changed At"@en ;
+   samm:description "Point in time when the content (any property according to the data model) of the capacity group was changed, at the supplier, either by a human user or an automated process."@en ;
+   samm:characteristic :Timestamp ;
+   samm:exampleValue "2023-03-10T12:27:11.320Z"^^xsd:dateTimeStamp .
+
+:linkedCapacityGroups a samm:Property ;
+   samm:preferredName "Linked Capacity Groups"@en ;
+   samm:description "A set of capacity groups linked to this capacity group."@en ;
+   samm:characteristic :LinkedCapacityGroupsCharacteristic ;
+   samm:exampleValue "be4d8470-2de6-43d2-b5f8-2e5d3eebf3fd" .
+
+:unitOfMeasureIsOmitted a samm:Property ;
+   samm:preferredName "Unit Of Measure Is Omitted"@en ;
+   samm:description "Explicit indicator of wether the unit of measure is left out of the payload intentionally. If \"true\" it means the sending application sends demand values without unit of measure intentionally and the unit of measure MUST NOT be contained in the payload. If \"false\" a unit of measure MUST be supplied."@en ;
```

Comment:
Why is this property necessary, when `unitOfMeasure` is already optional and allows to express exactly the same as this property? All "features" `unitOfMeasureIsOmitted` adds is an extra validation on the API level. Otherwise, it offers no benefit. If `unitOfMeasure` is missing in the payload, it obviously was omitted, no need for `unitOfMeasureIsOmitted`. If `unitOfMeasure` _is_ present in the payload, well, then it obviously was not omitted, and the extra property is also not necessary. I really don't see why we need this redundant property...

Reply:
As discussed in the architecture round, this property will stay in the model to make the payload without the UoM more speaking: Instead of just not having a unitOfMeasure property, which might also be the result of a faulty implementation, the additional "omitted" field makes it more semantically explicit, even though being technically redundant.